pkg/ring/kv/memberlist/metrics.go

```diff
@@ -17,7 +22,7 @@ func (m *Client) createAndRegisterMetrics() {
 	m.totalSizeOfReceivedMessages = prometheus.NewCounter(prometheus.CounterOpts{
 		Namespace: m.cfg.MetricsNamespace,
 		Subsystem: subsystem,
-		Name:      "received_broadcasts_size",
+		Name:      "received_broadcasts_bytes",
```

Comment:
A counter should have the `_total` suffix according to [Prometheus conventions](https://prometheus.io/docs/practices/naming/). Same comment applies to other counters in this file.

_I'm mentioning it because memberlist is still experimental, so introducing breaking changes in the metrics name shouldn't be an issue._

Reply:
> A counter should have the _total suffix according to Prometheus conventions. Same comment applies to other counters in this file.

Fixed (I don't like this specific convention at all).

> I'm mentioning it because memberlist is still experimental, so introducing breaking changes in the metrics name shouldn't be an issue.

That's why this PR is fixing it now.

Reply:
Should we change metrics in `tcp_transport.go` as well, to keep consistency?

Reply:
> Should we change metrics in `tcp_transport.go` as well, to keep consistency?

You're adding more salt to the wound now :) OK, ok, ...

Reply:
Unintentionally. I'm OK if you prefer to remove the `_total` suffix, but if we add it, we should do it in all memberlist metrics.